packages/components/focus-trap/src/utils.ts

```diff
@@ -81,15 +82,30 @@ export const tryFocus = (
 ) => {
   if (element && element.focus) {
     const prevFocusedElement = document.activeElement
+    let cleanup: boolean = false
+
+    if (
+      isElement(element) &&
+      !isFocusable(element) &&
+      !element.getAttribute('tabindex')
+    ) {
+      element.setAttribute('tabindex', '-1')
+      cleanup = true
+    }
+
     element.focus({ preventScroll: true })
     lastAutomatedFocusTimestamp.value = window.performance.now()
+
     if (
       element !== prevFocusedElement &&
       isSelectable(element) &&
       shouldSelect
     ) {
       element.select()
     }
+    if (isElement(element) && cleanup) {
```

Comment:
```suggestion
    if (cleanup) {
```
`cleanup` being `true` already satisfies `isElement(element)` being true.

Reply:
类型“{ focus: () => void; }”上不存在属性“removeAttribute”